test/test_user.py

```diff
@@ -25,7 +29,31 @@ def test_register(self, client, db_no_rollback):
         assert response_json['user']['email'] == data['email']
 
     @pytest.mark.run(order=12)
-    def test_login(self, client, db_no_rollback):
+    def test_register_duplicate_email(self, client, db_no_rollback):
+        data = {
+            'email': self.right_email,
+            'password': self.right_password
+        }
+        response = self.post(client, '/register', data)
+        assert response.status_code == 409
+        response_json = response.json()
+        assert response_json['success'] == 0
+        assert response_json['msg'] == error_msg.DUPLICATE_EMAIL
```

Comment:
Don't import other internal modules in the future.. this is kind of making things more complicated. Just asserting strings is fine. We're just doing black box tests here.